go/vt/vtgate/planbuilder/testdata/postprocess_cases.json

```diff
@@ -2088,16 +2088,16 @@
           {
             "OperatorType": "Aggregate",
             "Variant": "Ordered",
-            "Aggregates": "min(1|3) AS min(a.id)",
-            "GroupBy": "(0|2)",
-            "ResultColumns": 2,
+            "Aggregates": "any_value(2)",
+            "GroupBy": "(0|1)",
+            "ResultColumns": 3,
             "Inputs": [
```

Comment:
Filter above will receive random value of `min(a.id)` this looks unexpected, we should perform the min aggregation of min coming from the route below and then send that to the filter above.

Reply:
good catch!